src/components/listview/listview.js

```diff
@@ -426,7 +426,7 @@ define(['itemHelper', 'mediaInfo', 'indicators', 'connectionManager', 'layoutMan
 
             html += '<div class="' + cssClass + '">';
 
-            var moreIcon = '&#xE5D3;';
+            var moreIcon = '&#xe5d4';
```

Comment:
why do we use hex code here?

Reply:
No clue honestly. But I think that the main reason discussed in Matrix was for accessibility

Reply:
we should be using named classes, but whatever